test/unit-tests/function/geometry/distance.test.js

```diff
@@ -110,5 +140,6 @@ describe('distance', function () {
     assert.deepStrictEqual(bigmath.evaluate('distance([2, 3, 1], [1, 1, 2, 5, 0, 1])'), bignumber('2.3204774044612855517320588018918'))
```

Comment:
Just wondering (this test was already there): shouldn't this test throw an exeption because the two arrays don't have the same size? Or am I overlooking something?

Reply:
so there are multiple types of distance in this function for some reason. Euclidean distance of 2 points but also of points from a line done in multiple ways. Those sometimes have one extra value needed or twice as many values needed. It seemed odd to have these all in one function but I didnt question it.

Reply:
ahh, ok. Well, let's not change that functionality now then since that would create a breaking change.